docs/performance-scoring.md

```diff
@@ -0,0 +1,40 @@
+# Goal
+
+The goal of this document is to explain how performance scoring on Lighthouse works and what to do to improve your Lighthouse performance score. 
+
+# What performance metrics does Lighthouse measure?
+
+Lighthouse measures the following performance metrics: 
+
+- [First meaningful paint](https://developers.google.com/web/tools/lighthouse/audits/first-meaningful-paint): first meaningful paint is defined as when the browser first puts any “meaningful” element/set of “meaningful” elements on the screen. What is meaningful is determined from a series of heuristics. 
+- [First interactive](https://developers.google.com/web/tools/lighthouse/audits/first-interactive): first interactive is defined as the first point at which the page could respond quickly to input. It doesn't consider any point in time before first meaningful paint. The way this is implemented is primarily based on heuristics. 
+*Note: this metric is currently in beta, which means that the underlying definition of this metric is in progress.*
+- [Consistently interactive](https://developers.google.com/web/tools/lighthouse/audits/consistently-interactive): defined as the first point at which everything is loaded such that the page will quickly respond to any user input throughout the page. 
+*Note: this metric is currently in beta, which means that the underlying definition of this metric is in progress.*
+- [Perceptual Speed Index (pSI)](https://developers.google.com/web/tools/lighthouse/audits/speed-index): pSI measures how many pixels are painted at each given time interval on the viewport. The fewer the pixels that are painted, the better since we want an experience where most of the content is shown on the screen during the first few moments of initiating the page load. Therefore, the lower the pSI score, the better. 
```

Comment:
we'll want to update the document that this came from to adjust the "fewer the pixels that are painted, the better" part

It's not painting fewer pixels necessarily improves your score; it's that not painting pixels *late* is what improves your score. More generally, I'm wondering if we can provide more technical detail in the Lighthouse docs since the audience on GitHub is probably more savvy than the original audience of the doc?

Reply:
Done